src/NServiceBus.AcceptanceTests/ConfigureEndpointLearningPersistence.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting.Support;
+using NServiceBus.Persistence;
+
+public class ConfigureEndpointLearningPersistence : IConfigureEndpointTestExecution
+{
+    public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings, PublisherMetadata publisherMetadata)
+    {
+        storageDir = Path.Combine(@"c:\temp", Guid.NewGuid().ToString().Replace("-", "")); //can't use bindir since that will be to long on the build agents
```

Comment:
While "c:\temp" could be ensured to be there, should we lean to using `%TEMP%` with [`Environment.ExpandEnvironmentVariables()`](https://msdn.microsoft.com/en-us/library/system.environment.expandenvironmentvariables(v=vs.110).aspx)?